lbuild/module.py

```diff
@@ -396,6 +397,12 @@ def __init__(self,
         self.options = {}
 
     @property
+    def _logger(self):
+        if self.__logger is None and self._fullname is not None:
+            self.__logger = logging.getLogger(self._fullname)
```

Comment:
I think it's pretty clear, because the logger is named after the module. I could prefix this with `user:modm:platform:gpio`, but I'm not entirely sure what you mean with prefix?